AirportDictionary/AirportDictionary/Airport.cs

```diff
@@ -1,12 +1,6 @@
-﻿// <auto-generated />
-//
-// To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
-//
-//    using QuickTypeAirport;
-//
-//    var airport = Airport.FromJson(jsonString);
```

Comment:
Good to remove commented-out code.  It's considered bad practice to push commented-out code to VCS, because you can always go back in history to see what used to be there.